src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -439,6 +445,11 @@
 
             _hashChannel.Writer.Complete();
 
+            if (_ignoredFilesCount > 0)
```

Comment:
>Warnings.LogOneOrMoreFilesSkippedDueToSize(rootContext); [](http://example.com/codeflow?start=16&length=56)

@michaelcfanning question, do you think we should also change the same in file `AnalyzeCommandBase.cs` ?
If so I can work on the change together.

Reply:
For sure if we're missing this elsewhere, let's fix it. Here's a release note entry:

FEATURE: provide explicit logging (and new runtime condition flag) when any file is skipped due to exceeding file size limits.

Reply:
Got it, working on it.

Reply:
Synced with Mary, working on that internal BinSkim rule request first, will return to this once PR is published.